Create_climate_files_for_coordinates_of_intrest.ipynb

```diff
@@ -0,0 +1,208 @@
+{
```

Comment:
<p>you could add a link to https://cluster.klima.uni-bremen.de/~oggm/rgi/rgi62_stats.h5</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OGGM/oggm-shop-notebooks/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Thanks for the suggestion. I added the link :)